src/app/login/loginData.ts

```diff
@@ -0,0 +1,21 @@
+export class LoginData
+{
+    // Variables
+    private _mail: string;
+    private _password: string;
+
+    // Constructor
+    constructor(mail: string, password: string)
+    {
+        this._mail = mail;
+        this._password = password;
+
+    }
+
+    // Getters
+    get mail(): string { return this._mail; }
+    get password(): string { return this._password; }
+
+    // Setters
+
+}
```

Comment:
Instead of creating a class which describes login data, you can make this an interface instead.
An interface describes how data should look in an object.